packages/core/types/FsdtMessage.ts

```diff
@@ -10,8 +11,9 @@ export type FsdtServerMessage = FsdtMessage & {
 }
 
 export type FsdtLogMessage = {
-  log: string;
+  content: string;
   timestamp: number;
```

Comment:
is timestamp the best way to store datetime ?

Reply:
Do you have another idea?

Reply:
Maybe UTC date/time, timestand isn't easely readable. Everywhere we use it, we gonna have to reformat it. I'm ok with timestanp, I'm just thinking if it's the best choice

Reply:
Ok I fixed it